### DISCUSSION TITLE
What's the latest stable version?

### DISCUSSION THREAD

**Discussion by danardf** [fail2ban/fail2ban#3156] [Q&A] [answered]
Hi everyone.
I'd like to know what's the latest stable version.

On your Wiki page, this is the 0.9.4.
On git, we can see 0.11.x but we don't see the 0.11.x in your wiki page.

Can you help me please?

Thanks 
Franck

**Comment by sebres** [comment, marked as answer]
The answer is simple - fail2ban web page is not up-to-date (no time for that).
Current stable is 0.11.2 (released) and 0.11.3 (not released yet).

**Reply by danardf** [reply]
Oh ok, good to know.
That will be useful updating the wiki page, because we can get confuse. 
Thank you for your reply.
Have  a nice day.
